awx/ui_next/src/screens/User/User.test.jsx

```diff
@@ -24,29 +20,65 @@ async function getUsers() {
 }
 
 describe('<User />', () => {
-  test('initially renders succesfully', () => {
+  test('initially renders successfully', async () => {
     UsersAPI.readDetail.mockResolvedValue({ data: mockDetails });
     UsersAPI.read.mockImplementation(getUsers);
-    mountWithContexts(<User setBreadcrumb={() => {}} me={mockMe} />);
+    const history = createMemoryHistory({
+      initialEntries: ['/users/1'],
+    });
+    await act(async () => {
+      mountWithContexts(<User setBreadcrumb={() => {}} />, {
+        context: {
+          router: {
+            history,
+            route: {
+              location: history.location,
+              match: {
+                params: { id: 1 },
+                url: '/users/1',
+                path: '/users/1',
+              },
+            },
+          },
+        },
+      });
+    });
   });
 
   test('notifications tab shown for admins', async () => {
```

Comment:
The title of this test is misleading. The User card should display 5 tabs: Details, Organizations, Teams, Access, and Tokens. This test could be improved by asserting that those 5 tab titles are rendered.